core/controllers/base.py

```diff
@@ -295,8 +299,71 @@ def dispatch(self):
                 self.handle_exception(e, self.app.debug)
                 return
 
+        try:
+            self.validate_args_schema()
+        except Exception as e:
+            self.handle_exception(e, self.app.debug)
+            return
+
         super(BaseHandler, self).dispatch()
 
+    def validate_args_schema(self):
+        """Validates schema for controller layer handler class arguments.
+
+        Raises:
+            InvalidInputException. Schema validation failed.
+            NotImplementedError. Schema is not provided in handler class.
+        """
+        handler_class_name = self.__class__.__name__
+        request_method = self.request.environ['REQUEST_METHOD']
+        handler_args = self.payload
+        url_path_args = self.request.route_kwargs
+        non_schema_handlers = payload_validator.NON_SCHEMA_HANDLERS
+
+        if handler_class_name in non_schema_handlers:
+            return
+
+        if self.URL_PATH_ARGS_SCHEMAS is None:
+            raise NotImplementedError(
+                'Please provide schema for url path args in %s '
```

Comment:
Please provide --> Missing

(since these errors are typically directed to the end user)

Reply:
Done